jenkins/e2e-image/e2e-runner.sh

```diff
@@ -84,6 +84,54 @@ function running_in_docker() {
     grep -q docker /proc/self/cgroup
 }
 
+### FUNCTIONS TO SUPPORT get-kube.sh ###
+
+# Moves local tarballs into a local path suitable for get-kube.sh, and sets
+# KUBERNETES_RELEASE and KUBERNETES_RELEASE_URL accordingly.
+function set_release_vars_from_local_gcs_stage() {
```

Comment:
The net effect of my change:
* we no longer pull tarballs from GCS in `e2e-runner.sh`
* we no longer extract tarballs in `e2e-runner.sh`